repo/packages/M/marathon-lb/19/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "2.0",
```

Comment:
I'm not sure why this is still version 2.0.  this is very old.  version 3.0 is the latest and we are moving to 4 soon.